player/command.c

```diff
@@ -2823,28 +2823,50 @@ static int mp_property_focused(void *ctx, struct m_property *prop,
     return m_property_bool_ro(action, arg, focused);
 }
 
+static int get_display_names_subkey(struct vo *vo, int action,
+                                    void *arg, enum mp_voctrl voctrl)
+{
+    switch (action) {
+        case M_PROPERTY_GET_TYPE:
+            *(struct m_option *)arg = (struct m_option){.type = CONF_TYPE_STRING_LIST};
+            return M_PROPERTY_OK;
+        case M_PROPERTY_GET: {
+            char** display_names;
+            if (vo_control(vo, voctrl, &display_names) < 1)
+                return M_PROPERTY_UNAVAILABLE;
+
+            *(char ***)arg = display_names;
+            return M_PROPERTY_OK;
+        }
+    }
+    return M_PROPERTY_NOT_IMPLEMENTED;
+}
+
 static int mp_property_display_names(void *ctx, struct m_property *prop,
                                      int action, void *arg)
 {
-    MPContext *mpctx = ctx;
-    struct vo *vo = mpctx->video_out;
+    MPContext *const mpctx = ctx;
+    struct vo *const vo = mpctx->video_out;
```

Comment:
I don't disagree, but we generally don't do that, for simplicity, those consts don't change anything here.